src/components/index/BlockchainFeaturesSection.tsx

```diff
@@ -102,7 +102,6 @@ export function BlockchainFeaturesSection(): JSX.Element {
           </Slider>
         </div>
       </Container>
-      <div className="hidden md:block absolute h-[785.44px] w-[785.44px] md:left-[-240px] md:bottom-0 lg:left-0 lg:bottom-[-289.56px] bg-contain bg-no-repeat mix-blend-screen bg-[url('/assets/img/blockchainFeatures/blockchainFeaturesBg.png')]" />
```

Comment:
maybe reconsider not removing this due to this [comment](https://github.com/DeFiCh/defichain.com/pull/505#issuecomment-1471342813)?